src/Navigation/Search/searchSaga.js

```diff
@@ -1,17 +1,17 @@
 import { put, call, debounce } from "redux-saga/effects";
 import { fetchDataSucces, fetchDataFailure, fetchData } from "./searchSlice";
-import { getSearch } from "./getSearch";
+import { getSearchMovie } from "./getSearch";
 
 function* fetchDataHandler(action) {
     try {
         yield call(fetchData);
-        const result = yield call(getSearch, action.payload.query);
+        const result = yield call(getSearchMovie, action.payload.query);
         yield put(fetchDataSucces({ data: result }));
```

Comment:
👍👍 maybe we should use that in all sagas - no need to put whole response in store, just the value if result prop 🧠